src/routes/users.rs

```diff
@@ -121,6 +121,38 @@ pub async fn mods_list(
     }
 }
 
+#[get("invites")]
+pub async fn remove_team_member(
+    req: HttpRequest,
+    info: web::Path<(UserId,)>,
+    pool: web::Data<PgPool>,
+) -> Result<HttpResponse, ApiError> {
+    let id: crate::database::models::UserId = info.into_inner().0.into();
+
+    let current_user = get_user_from_headers(req.headers(), &**pool).await.map_err(|_| ApiError::AuthenticationError)?;
+
+    let mut results = Vec::new();
+
+    let id_int = id.0 as u64;
+    if current_user.id.0 == id_int {
+        results = crate::database::models::TeamMember::get_from_user_private(id, &**pool).await?;
```

Comment:
This returns a list of all team members of a user, both accepted and not accepted.  If this is meant to list unaccepted invites, this should skip the accepted ones.